co/dataOStream.h

```diff
@@ -214,7 +214,10 @@ class DataOStream : public boost::noncopyable
 
     /** Write a non-plain data item. */
     template< class T > void _write( const T& value, const boost::false_type& )
-    { _writeSerializable( value, boost::is_base_of< servus::Serializable, T >());}
+    {
+        _writeSerializable( value,
+                            boost::is_base_of< servus::Serializable, T >( ));
```

Comment:
Doesn't it make more sense to have an overload of _write for servus::Serializable instead of doing template metaprograming.